examples/advanced/job-level-authorization/jobs/job1/meta.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "hello-numpy-sag",
+  "resource_spec": {},
+  "min_clients" : 1,
+  "deploy_map": {
+    "app": [
+      "@ALL"
+    ]
+  }
+}
```

Comment:
the job creation and these meta.json replacement could be done using the new JOB CLI, we can enhance this in the next PR

Reply:
we can enhance these points in follow up PR.